configure.ac

```diff
@@ -61,6 +61,8 @@ if test "$fhs" = "yes"; then
     execcgidir=$libdir/sympa/cgi
     expldir=$localstatedir/lib/sympa/list_data
     staticdir=$localstatedir/lib/sympa/static_content
+    cssdir=$dlocalestatedir/lib/sympa/static_content/css
+    picturesdir=$dlocalestatedir/lib/sympa/static_content/pictures
```

Comment:
s/dlocalestatedir/localestatedir/

Reply:
Ouch, comment has typo.

s/dlocalestatedir/localstatedir/

Reply:
Oops, sorry for the silo typo :-(
I fixed the commit and force-pushed.